multimodal/src/autogluon/multimodal/data/utils.py

```diff
@@ -421,6 +432,39 @@ def is_rois_input(sample):
     return isinstance(sample, list) and len(sample) and isinstance(sample[0], list) and len(sample[0]) == 5
 
 
+def is_image_input(images, image_type):
```

Comment:
We can remove this util since it's used only in one place.

Reply:
Reminder: consider removing this util and reverting the related changes.

Reply:
Done.